src/UI/styles/autofill-tooltip-styles.css

```diff
@@ -10,15 +10,13 @@
     top: 0;
     left: 0;
     padding: 0;
-    font-family: 'DDG_ProximaNova', 'Proxima Nova', -apple-system,
-    BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+    font-family: 'DDG_ProximaNova', 'Proxima Nova', system-ui, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
     'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
     -webkit-font-smoothing: antialiased;
     z-index: 2147483647;
 }
-:not(.top-autofill).wrapper--data {
-    font-family: 'SF Pro Text', -apple-system,
-    BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+.wrapper--data {
+    font-family: 'SF Pro Text', system-ui, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
```

Comment:
This shouldn't have been restricted to `.top-autofill`. Also added `system-ui` as it's supported on all platforms.